addons/bus/static/src/workers/websocket_worker.js

```diff
@@ -214,8 +216,13 @@ export class WebsocketWorker {
      *     - Number: user is logged whether on the frontend/backend.
      *     - false: user is not logged.
      *     - undefined: not available (e.g. livechat support page)
+     * @param {Number} param0.startTs Timestamp of start of bus service sender.
      */
-    _initializeConnection(client, { debug, lastNotificationId, uid, websocketURL }) {
+    _initializeConnection(client, { debug, lastNotificationId, uid, websocketURL, startTs }) {
+        if (this.newestStartTs && this.newestStartTs > startTs) {
+            return;
```

Comment:
In this case, the `bus_service` will never start and the webclient will be blocked forever. This is due to the bus service awaiting an acknowledgment for the worker initialization. If `this.newStartTs` is > to `startTs` this method should still: 
- update the debug mode (still relevant)
- send the `initialized` ack to the client.

Reply:
done